<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="FirstChanceExceptionEventArgs.xml" source-language="en-US" target-language="de-DE">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac55b82b33fd434c2b4d5434a6080b2c61d46168149.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">5b82b33fd434c2b4d5434a6080b2c61d46168149</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netcore-2.0,netcore-2.1,netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7,netstandard-2.0,xamarinandroid-7.1,xamarinios-10.8,xamarinmac-3.0</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs">
          <source>Provides data for the notification event that is raised when a managed exception first occurs, before the common language runtime begins searching for event handlers.</source>
          <target state="translated">Stellt Daten für das Benachrichtigungsereignis bereit, das beim erstmaligen Auftreten einer verwalteten Ausnahme ausgelöst wird, bevor die Common Language Runtime beginnt, nach Ereignishandlern zu suchen.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs">
          <source>This class provides the exception handler for the <ph id="ph1">&lt;xref:System.AppDomain.FirstChanceException?displayProperty=nameWithType&gt;</ph> event with access to the exception.</source>
          <target state="translated">Diese Klasse stellt den Ausnahmehandler für die <ph id="ph1">&lt;xref:System.AppDomain.FirstChanceException?displayProperty=nameWithType&gt;</ph> Ereignis "mit" Zugriff auf die Ausnahme.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs">
          <source>The following example creates a series of application domains named <ph id="ph1">`Child_0`</ph> through <ph id="ph2">`Child_3`</ph>, with a <ph id="ph3">`Worker`</ph> object in each application domain.</source>
          <target state="translated">Das folgende Beispiel erstellt eine Reihe von Anwendungsdomänen, die mit dem Namen <ph id="ph1">`Child_0`</ph> über <ph id="ph2">`Child_3`</ph>, mit einem <ph id="ph3">`Worker`</ph> Objekt pro Anwendungsdomäne.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs">
          <source>Each <ph id="ph1">`Worker`</ph> object has a reference to the <ph id="ph2">`Worker`</ph> object in the next application domain, except for the <ph id="ph3">`Worker`</ph> in the last application domain.</source>
          <target state="translated">Jede <ph id="ph1">`Worker`</ph> Objekt enthält einen Verweis auf die <ph id="ph2">`Worker`</ph> -Objekt in der nächsten Anwendungsdomäne, außer für die <ph id="ph3">`Worker`</ph> in der letzten Anwendungsdomäne.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs">
          <source>The <ph id="ph1">&lt;xref:System.AppDomain.FirstChanceException&gt;</ph> event is handled in all application domains except <ph id="ph2">`Child_1`</ph>.</source>
          <target state="translated">Die <ph id="ph1">&lt;xref:System.AppDomain.FirstChanceException&gt;</ph> Ereignis behandelt wird, in allen Anwendungsdomänen außer <ph id="ph2">`Child_1`</ph>.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs">
          <source>When the application domains have been created, the default application domain calls the <ph id="ph1">`TestException`</ph> method for the first child application domain.</source>
          <target state="translated">Wenn die Anwendungsdomänen erstellt wurden, ruft die Standardanwendungsdomäne die <ph id="ph1">`TestException`</ph> Methode für die erste untergeordnete Anwendungsdomäne.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs">
          <source>Each <ph id="ph1">`Worker`</ph> object calls the <ph id="ph2">`TestException`</ph> method for the next, until the last <ph id="ph3">`Worker`</ph> throws an exception that is either handled or unhandled.</source>
          <target state="translated">Jede <ph id="ph1">`Worker`</ph> -Objekt ruft die <ph id="ph2">`TestException`</ph> Methode für die nächsten, bis der letzte <ph id="ph3">`Worker`</ph> löst eine Ausnahme, die entweder behandelt oder nicht behandelt wird.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs">
          <source>Thus, the current thread passes through all the application domains, and <ph id="ph1">`TestException`</ph> is added to the stack in each application domain.</source>
          <target state="translated">Folglich alle Anwendungsdomänen, der aktuelle Thread durchläuft und <ph id="ph1">`TestException`</ph> wird dem Stapel in jeder Anwendungsdomäne hinzugefügt.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs">
          <source>When the last <ph id="ph1">`Worker`</ph> object handles the exception, the <ph id="ph2">&lt;xref:System.AppDomain.FirstChanceException&gt;</ph> event is raised only in the last application domain.</source>
          <target state="translated">Wenn der letzte <ph id="ph1">`Worker`</ph> Objekt verarbeitet die Ausnahme, die <ph id="ph2">&lt;xref:System.AppDomain.FirstChanceException&gt;</ph> Ereignis wird nur in der letzten Anwendungsdomäne.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs">
          <source>The other application domains never get a chance to handle the exception, so the event is not raised.</source>
          <target state="translated">Die anderen Anwendungsdomänen Gelegenheit nie zur Behandlung von Ausnahmen, damit das Ereignis nicht ausgelöst wird.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs">
          <source>When the last <ph id="ph1">`Worker`</ph> object does not handle the exception, the <ph id="ph2">&lt;xref:System.AppDomain.FirstChanceException&gt;</ph> event is raised in each application domain that has an event handler.</source>
          <target state="translated">Bei der letzten <ph id="ph1">`Worker`</ph> Objekt behandelt die Ausnahme nicht die <ph id="ph2">&lt;xref:System.AppDomain.FirstChanceException&gt;</ph> Ereignis wird in jeder Anwendungsdomäne, die einen Ereignishandler aufweist.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs">
          <source>After each event handler has finished, the stack continues to unwind until the exception is caught by the default application domain.</source>
          <target state="translated">Nach Abschluss der einzelnen Ereignishandler wird der Stapel entladen, bis die Ausnahme durch die Standardanwendungsdomäne abgefangen wird fortgesetzt.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs">
          <source>To see how the stack display grows as the event is raised closer and closer to the default application domain, change <ph id="ph1">`e.Exception.Message`</ph> to <ph id="ph2">`e.Exception`</ph> in the <ph id="ph3">`FirstChanceHandler`</ph> event handlers.</source>
          <target state="translated">Um anzuzeigen, wie die Anzeige der Stapel vergrößert wird, wie das Ereignis, das näher ausgelöst wird und näher an die Standardanwendungsdomäne, ändern Sie <ph id="ph1">`e.Exception.Message`</ph> auf <ph id="ph2">`e.Exception`</ph> in der <ph id="ph3">`FirstChanceHandler`</ph> -Ereignishandler.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs">
          <source>Notice that when <ph id="ph1">`TestException`</ph> is called across application domain boundaries, it appears twice: once for the proxy and once for the stub.</source>
          <target state="translated">Beachten Sie, dass bei <ph id="ph1">`TestException`</ph> heißt über Anwendungsdomänengrenzen hinweg, dann zweimal angezeigt: einmal für den Proxy und einmal für den Stub.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" uid="M:System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs.#ctor(System.Exception)">
          <source>The exception that was just thrown by managed code, and that will be examined by the <ph id="ph1">&lt;see cref="E:System.AppDomain.UnhandledException" /&gt;</ph> event.</source>
          <target state="translated">Die Ausnahme, die gerade von verwaltetem Code ausgelöst wurde und die vom <ph id="ph1">&lt;see cref="E:System.AppDomain.UnhandledException" /&gt;</ph>-Ereignis untersucht wird.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" uid="M:System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs.#ctor(System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs" /&gt;</ph> class with a specified exception.</source>
          <target state="translated">Initialisiert eine neue Instanz der <ph id="ph1">&lt;see cref="T:System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs" /&gt;</ph>-Klasse mit einer angegebenen Ausnahme.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs.#ctor(System.Exception)">
          <source>This constructor is called by the common language runtime, not by application code.</source>
          <target state="translated">Dieser Konstruktor wird von der common Language Runtime nicht von Anwendungscode aufgerufen.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" uid="P:System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs.Exception">
          <source>The managed exception object that corresponds to the exception thrown in managed code.</source>
          <target state="translated">Das verwaltete Ausnahmeobjekt, das der ausgelösten Ausnahme in verwaltetem Code entspricht.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs.Exception">
          <source>The newly thrown exception.</source>
          <target state="translated">Die gerade ausgelöste Ausnahme.</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs.Exception">
          <source>The method that handles the <ph id="ph1">&lt;xref:System.AppDomain.UnhandledException?displayProperty=nameWithType&gt;</ph> event can examine the exception and perform appropriate logging or other notification services.</source>
          <target state="translated">Die Methode, verarbeitet die <ph id="ph1">&lt;xref:System.AppDomain.UnhandledException?displayProperty=nameWithType&gt;</ph> Ereignis kann die Ausnahme untersuchen und eine entsprechende Protokollierung oder andere Notification Services ausführen.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs.Exception">
          <source>For example code, see <ph id="ph1">&lt;xref:System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs&gt;</ph>.</source>
          <target state="translated">Beispielcode finden Sie unter <ph id="ph1">&lt;xref:System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs&gt;</ph>.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>